hr_timesheet_holiday/models/analytic_account.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Sunflower IT <http://sunflowerweb.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class AnalyticAccount(models.Model):
+    """Add 'is leave account' flag to Analytic Account"""
+    _inherit = 'account.analytic.account'
+
+    is_leave_account = fields.Boolean(
```

Comment:
@thomaspaulb this field cannot be set from the UI currently - this way it's impossible to make an existing account a vacation account. Can you add that please?

Reply:
sorry, ignore. Temporary blindness